### PR TITLE
Add NodeJS 6 and Python for serverless triggers gcloud functions

### PR DESCRIPTION
**Pull request by rosariopfernandes**
### Description

Add NodeJS 6 and Python for serverless-triggers on google cloud functions.

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [x] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Related Issue

Related to #580 

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [x] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by rosariopfernandes** [comment]
@shahidhk Thanks for the feedback. I've changed it accordingly.

File: community/boilerplates/serverless-triggers/google-cloud-functions/python/echo/main.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import sys
+import json
+from flask import jsonify
+
+
+def function(request):
+    request_json = request.get_json
```
Comment (shahidhk) [inline community/boilerplates/serverless-triggers/google-cloud-functions/python/echo/main.py]: `get_json` is a function

**Comment by rosariopfernandes** [comment]
@shahidhk Just tested, it wasn't returning the JSON. I made some changes, and now it works. Please check.

**Comment by shark-h** [comment]
Hey @rosariopfernandes ! Can you please fill this [form](https://goo.gl/forms/4NxUyIeTVwyWjGqE3) so that we can send you some swag? 😄

**Comment by rosariopfernandes** [comment]
That's awesome! Just filled, Thanks @shark-h !

**Pull request merged**